zyte_common_items/components.py

```diff
@@ -127,6 +128,25 @@ class Link(Item):
     )
 
 
+@attrs.define(kw_only=True)
+class Anchor(Item):
```

Comment:
Anchor makes me think of the targets of URL fragments (#foo).

Maybe something like `NamedLink` would work?